data/pathfinder_pfs/paizo/pathfinder_society/pfs_chronicle_sheets/pfscs_abilitycategories.lst

```diff
@@ -11,208 +10,11 @@ ABILITYCATEGORY:PFSChronicle							VISIBLE:YES		EDITABLE:YES								CATEGORY:PFS
 ABILITYCATEGORY:Ally of Belhaim Perform Choice				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO		FRACTIONALPOOL:NO		CATEGORY:Special Ability	TYPE:AllyOfBelhaimPerformChoice																										DISPLAYLOCATION:Chronicles
 
 # These ability pools hold the chronicle sheet boons
-ABILITYCATEGORY:Chronicle_S00-01						VISIBLE:QUALIFY	EDITABLE:YES								CATEGORY:Special Ability	TYPE:Chronicle_S00-01											DISPLAYNAME:PFS Scenario #01: Silent Tide											DISPLAYLOCATION:Chronicles
```

Comment:
When I see large chunks of data removed, it makes me nervous. What is the reasoning for this?